runtime/Makefile

```diff
@@ -1,25 +1,33 @@
 PYTHON := python3
 C_COMPILER?=clang
 CXX_COMPILER?=clang++
-COMPILER_LAUNCHER?=ccache
-ENABLE_ASAN?=OFF
+COMPILER_LAUNCHER?=$(shell which ccache)
+
 MK_ABSPATH := $(abspath $(lastword $(MAKEFILE_LIST)))
 MK_DIR := $(dir $(MK_ABSPATH))
 RT_BUILD_DIR?=$(MK_DIR)/build
 CMAKE_ARGS?=""
 CODE_COVERAGE?=OFF
-BUILD_TYPE?=Release
+BUILD_TYPE?=RelWithDebInfo
```

Comment:
Why change the default build type?

Reply:
I realized building with debug info is really a great help when it comes to debugging and fixing runtime-related bugs.

Reply:
I remove it from the PR. Let's revisit this in a separate PR to check its overall impact on the wheels.